src/main/scala/nl.knaw.dans.easy.sword2/DepositHandler.scala

```diff
@@ -106,12 +106,7 @@ object DepositHandler extends BagValidationExtension {
     // to ensure all files in deposit are accessible for the deposit group, the method setFilePermissions is always
     // executed (regardless of whether extractAndValidatePayloadAndGetDepositReceipt was successful or not).
     // Otherwise operators don't have the proper permissions to clean or fix the invalid zip files or bags that might stay behind
-    extractAndValidatePayloadAndGetDepositReceipt(deposit, contentLength, payload, depositDir) match {
```

Comment:
@rvanheest : do you remember if posponing the change-permission-step until after handing off the deposit to the finalizing thread had a particular reason. This construction below seems a bit convoluted, anyway.